cognite_toolkit/cdf_tk/load.py

```diff
@@ -403,16 +406,42 @@ def get_required_capability(cls, ToolGlobals: CDFToolConfig) -> Capability:
     def get_id(cls, item: Group) -> str:
         return item.name
 
-    def load_file(self, filepath: Path, ToolGlobals: CDFToolConfig) -> Group:
+    def replace_scope_ids(self, capability: dict, ToolGlobals: CDFToolConfig):
+        if len(capability.get("scope", {}).get("datasetScope", {}).get("ids", [])) > 0:
+            capability["scope"]["datasetScope"]["ids"] = [
+                ToolGlobals.verify_dataset(ext_id)
+                for ext_id in capability.get("scope", {}).get("datasetScope", {}).get("ids", [])
+            ]
+
+        if len(capability.get("scope", {}).get("extractionPipelineScope", {}).get("ids", [])) > 0:
+            capability["scope"]["extractionPipelineScope"]["ids"] = [
+                ToolGlobals.verify_extraction_pipeline(ext_id)
```

Comment:
Here we get into the same issue as I had in #180 if we want to do a clean command and the target scope does not exist. We want to be able to load all extraction pipelines, also the ones where we don't have the scope stored in the CDF project as we need a complete list to be used for deletion (a later pipeline may be present and should be deleted).